babel.cfg

```diff
@@ -0,0 +1,2 @@
+[python: **.py]
+[jinja2: **/templates/**.html]

```

Comment:
Are you sure the file is needed?

Reply:
The templates dir isn't really needed. But we must have the html files. Otherwise we won't get the translations of them via Babel